frontend/src/components/about/Elixir.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <about-layout title="ELIXIR">
+    <template v-slot:contents>
+      <p>Metabolic Atlas would not be possible without the following excellent Elixir resources:</p>
+      <br />
+      <div v-for="link in links" :key="link.name" class="columns is-vcentered">
+        <br />
+        <div class="column is-1">
+          <a :href="link.link" target="_blank">
+            <template v-if="link.icon">
+              <img :src="link.icon" />
+            </template>
+          </a>
+        </div>
+        <div class="text-column column has-text-justified">
```

Comment:
I'm not sure what was intended here
- if more space is needed between columns, please use https://bulma.io/documentation/columns/gap/
- if more padding is needed inside a column, please use https://bulma.io/documentation/helpers/spacing-helpers/

Reply:
Just to clarify, you mean the `text-column` and the `has-text-justified` classes right?

Reply:
Ok, I actually mentioned something about this in this PR description, see attached image for example. We can leave things as it is in your new PR for now and discuss this more in the future.

> Removed justified attribute for text. Justified text results in inconsistent spacing between words and the spacing can become excessively wide especially in smaller screens. Both this and the previous point are to help increase readability.

<img width="480" alt="image" src="https://user-images.githubusercontent.com/423498/153202968-9b5ef35b-3b6b-4204-ac60-f39d316953eb.png">

Reply:
Interesting, the screenshot you're sharing is from `Related resources` whereas my comment is for the `Elixir` page.
Wouldn't a Bulma-defined `pr-3` class be sufficient, instead of defining a custom class?

Reply:
Ah sorry, I was working on the `Related resources` page and got confused. Point still applies though about my opinion on avoiding `has-text-justified`, which is also used here. I have no argument about the `text-column` class and support your removal of it in the new PR.